lib/chef/resource/build_essential.rb

```diff
@@ -134,11 +135,23 @@ class BuildEssential < Chef::Resource
         #
         # @return [true, false]
         def xcode_cli_installed?
-          cmd = Mixlib::ShellOut.new("pkgutil --pkgs=com.apple.pkg.CLTools_Executables")
-          cmd.run_command
+          cmd = shell_out("pkgutil", "--pkgs=com.apple.pkg.CLTools_Executables")
           # pkgutil returns an error if the package isn't found aka not installed
           cmd.error? ? false : true
         end
+
+        #
+        # Return to package label of the latest XCode Command Line Tools update, if available
+        #
+        # @return [String]
+        def xcode_cli_package
+          cmd = <<-EOH.gsub(/^ {14}/, "")
```

Comment:
```suggestion
          cmd = <<-EOH.strip
```



Removing exactly 14 spaces from the front of this seems too specific (and also doesn't seem like it'll do anything here?). Using `.strip` will continue working if this code gets reindented, etc. There is also `.lstrip` if we want ot keep whitespace on the right side for whatever reason.

Reply:
There's one more occurrence of this in this file but I can't comment on it because it's not close enough to any lines modified in this PR.